src/cromshell/status/command.py

```diff
@@ -0,0 +1,123 @@
+import logging
+import click
+
+import requests
+import tempfile
+import json
+import re
+import fileinput
+import jq
+
+from cromshell.utilities import HTTPUtils
+from cromshell.utilities import IOUtils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="status")
+@click.option(
```

Comment:
do you think this is an option that you'd want to expose specifically for this `status` command?

Reply:
done, moved to main file and part of cromshell_config